eng/pipelines/eslint-plugin-regression.yml

```diff
@@ -0,0 +1,23 @@
+pr:
```

Comment:
Can we put this pipeline into the same directory as the eslint tool and rename it to ci.yml like our other pipelines?